src/java.desktop/share/classes/sun/font/StandardGlyphVector.java

```diff
@@ -1815,6 +1830,19 @@ void appendGlyphOutline(int glyphID, GeneralPath result, float x, float y) {
             PathIterator iterator = gp.getPathIterator(null);
             result.append(iterator, false);
         }
+
+        void appendGlyphRenderData(int glyphID, GlyphRenderData result, float x, float y) {
+            // !!! fontStrike needs a method for this.  For that matter, GeneralPath does.
```

Comment:
It's better to use well known "TODO" here

Reply:
That was just copied from `appendGlyphOutline` above :)